infrastructure/zk/src/hyperchain_wizard.ts

```diff
@@ -567,6 +575,67 @@ export function getTokens(network: string): L1Token[] {
     }
 }
 
-export const initHyperchainCommand = new Command('init-hyperchain')
-    .description('Initializes a new Hyperchain network')
+async function selectHyperchainConfiguration() {
+    const envs = env.getAvailableEnvsFromFiles();
+
+    const envQuestions = [
+        {
+            message: 'Which Hyperchain configuration do you want to use?',
+            name: 'env',
+            type: 'select',
+            choices: [...envs].sort()
+        }
+    ];
+
+    const envResults: any = await enquirer.prompt(envQuestions);
+    return envResults.env;
+}
+
+async function generateDockerImages() {
+    await warning(`\nThis process will build the docker images and it can take a while. Please be patient.\n`);
+
+    const envName = await selectHyperchainConfiguration();
+
+    await docker.customBuildForHyperchain('server-v2', envName);
+
+    await warning(`\n\n\n\n\nDocker image for server created: Server image: ${envName}/server-v2:latest\n`);
+
+    let hasProver = false;
+
+    if (process.env.ETH_SENDER_SENDER_PROOF_SENDING_MODE !== 'SkipEveryProof') {
+        hasProver = true;
+        // TODO: Hyperchain is using prover, so we must include Boojum images - wait for Boojum merge
+        // proof-fri-compressor, prover-fri, witness-generator, prover-fri-gateway
+    }
+
+    const composeArgs = {
+        envFilePath: `./etc/env/${envName}.env`,
+        serverImageName: envName,
+        hasProver
+    };
+
+    const templateFileName = 'docker-compose-hyperchain-template';
+
+    const templateString = fs.existsSync(templateFileName) && fs.readFileSync(templateFileName).toString().trim();
```

Comment:
The spacing feels a bit odd here, why an empty line between seemingly related lines?

Reply:
No good reason at all. Will change